README.md

```diff
@@ -1,45 +1,36 @@
-6.	You should immediately see a sphere with a ring on it appear. You should be able to interact with it. If you press the "W" key, this should jump you up close to a transit terminus on the ring. However, at this point the earth, and other objects, are missing its texture.
-7. To fix this (on Windows), enter the following commands...
+- **Setup**
 ```
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing> cd textures
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> wget https://www.project-atlantis.com/wp-content/threejs-simulation/textures/bluemarble_16384.png -outfile bluemarble_16384.png
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> curl.exe -O https://www.project-atlantis.com/wp-content/threejs-simulation/textures/moon.jpg
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> curl.exe -O https://www.project-atlantis.com/wp-content/threejs-simulation/textures/movingRingTexture.jpg
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> curl.exe -O https://www.project-atlantis.com/wp-content/threejs-simulation/textures/myakka_oli_2022031_lrg.jpg
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> mkdir 24x12\LR
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> mkdir 24x12\HR
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures> cd 24x12\LR
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures\24x12\LR> foreach ($X in @(0..23)) {foreach ($Y in @(0..11)) {curl.exe -O https://www.project-atlantis.com/wp-content/threejs-simulation/textures/24x12/LR/earth_LR_24x12_${X}x${Y}.jpg}}
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures\24x12\LR> cd ..\HR
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures\24x12\HR> foreach ($X in @(0..23)) {foreach ($Y in @(0..11)) {curl.exe -O https://www.project-atlantis.com/wp-content/threejs-simulation/textures/24x12/HR/earth_HR_24x12_${X}x${Y}.jpg}}
-PS C:\Users\Username\Documents\repos\Three.js\TetheredRing\textures\24x12\HR> cd ..\..\..
+cd TetheredRing
+npm install
+npm run dev-sync
+npm run dev
 ```
-8. Now if you go back to the browser tab, and press \<cntrl\> + \<shift\> + 'R', this will reload the webpage. The model may take longer to load, but now the earth should appear tectured.
+- **Open web browser to** `http://localhost:3000`
 
-If you make changes to main.js, or any of the other project files, these changes will immediately take effect in the browser. Note that when working on the project, on occasion you may need to refresh the browser with \<cntrl\> + \<shift\> + 'R' in order for the model to display properly.
+**Note:** If you make changes to main.js, or any of the other project files, these changes will immediately take effect in the browser. Note that when working on the project, on occasion you may need to refresh the browser with \<cntrl\> + \<shift\> + 'R' in order for the model to display properly.  
```

Comment:
There seems to be a new bug in node/vite that causes this statement not to be true: "If you make changes to main.js, or any of the other project files, these changes will immediately take effect in the browser." - at least when changing transitsystem.js.
```suggestion
**Note:** If you make changes to main.js, or any of the other project files, these changes will immediately take effect in the browser. (Warning: At the time this was written, there appears to be a bug in Vite that invalidates this statement for some files.)
**Note:** that when working on the project, on occasion you may need to refresh the browser with \<cntrl\> + \<shift\> + 'R' in order for the model to display properly.  
```